### PR TITLE
add kube-aggregator to hyperkube

### PR DESCRIPTION
**Pull request by deads2k**
Adds kube-aggregator to hyperkube.

@lavalamp @pwittrock @kubernetes/sig-api-machinery-misc @smarterclayton 

Google publishes a hyperkube image somewhere, right?

### CODE REVIEW

**Comment by deads2k** [comment]
@k8s-bot gci gce e2e test this

File: cmd/hyperkube/main.go [hunk 1]
```diff
@@ -43,5 +43,8 @@ func main() {
 	hk.AddServer(NewFederationAPIServer())
 	hk.AddServer(NewFederationCMServer())
 
+	// kube-aggregator
```
Comment (sttts) [inline cmd/hyperkube/main.go]: no comment, move up

File: staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go [hunk 2]
```diff
@@ -57,13 +57,7 @@ type AggregatorOptions struct {
 
 // NewCommandStartMaster provides a CLI handler for 'start master' command
```
Comment (sttts) [inline staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go]: NewCommandStartAggregator

**Comment by sttts** [comment]
Some nits. Otherwise, lgtm

**Comment by deads2k** [comment]
@smarterclayton bump

**Comment by smarterclayton** [comment]
/approve
/lgtm

after stefan

**Comment by deads2k** [comment]
@k8s-bot verify test this

**Pull request merged**